src/schemas/28-8832/schema.js

```diff
@@ -0,0 +1,119 @@
+import constants from '../../common/constants';
+import _ from 'lodash';
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'EDUCATIONAL/VOCATIONAL COUNSELING APPLICATION (28-8832)',
+  type: 'object',
+  additionalProperties: false,
+  definitions: _.pick(definitions, [
+    'fullName',
+    'address',
+    'phone',
+    'ssn',
+    'date',
+    'dateRange',
+    'gender'
+  ]),
+  properties: {
+    applicantFullName: {
+      $ref: '#/definitions/fullName'
+    },
+    applicantSocialSecurityNumber: {
+      $ref: '#/definitions/ssn'
+    },
+    applicantGender: {
+      $ref: '#/definitions/gender'
+    },
+    applicantEmail: {
+      type: 'string',
+      format: 'email'
+    },
+    applicantDateOfBirth: {
+      $ref: '#/definitions/date'
+    },
+    applicantRelationshipToVeteran: {
+      type: 'string'
+    },
+    applicantHomePhone: {
+      $ref: '#/definitions/phone'
+    },
+    applicantMobilePhone: {
+      $ref: '#/definitions/phone'
+    },
+    applicantAddress: {
+      $ref: '#/definitions/address'
+    },
+    seekingRestorativeTraining: {
+      type: 'boolean'
+    },
+    seekingVocationalTraining: {
+      type: 'boolean'
+    },
+    receivedPamphlet: {
+      type: 'boolean'
+    },
+    veteranFullName: {
+      $ref: '#/definitions/fullName'
+    },
+    veteranSocialSecurityNumber: {
+      $ref: '#/definitions/ssn'
+    },
+    veteranDateOfBirth: {
+      $ref: '#/definitions/date'
+    },
+    veteranServiceBranch: {
+      type: 'string'
+    },
+    veteranDateofDeathMIAPOW: {  // date of death, M.I.A., or P.O.W.
+      $ref: '#/definitions/date'
+    },
+    divorceOrAnnulmentPending: {
+      type: 'boolean',
+    },
+    applicantAgeAtRemarriage: {
+      type: 'number',
+    },
+    previousBenefitApplications: {
+      type: 'array',
+      items: {
+        type: 'string'
```

Comment:
I don't think we have a way to represent a list of strings schema like this on the front end. But we can leave until we know what the UI is supposed to look like.

Reply:
I wasn't quite sure how this would be getting mapped either, so it's more of a placeholder.